ydb/core/tx/columnshard/blob_manager.h

```diff
@@ -90,7 +90,8 @@ class IBlobManager {
     }
 
     // Deletes the blob that was previously permanently saved
```

Comment:
It seems to me that comment relates to old DeleteBlob and we can write new comments for DeleteBlobOnExecute and DeleteBlobOnComplete (prefer) or remove old comment.